lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/StakePools.hs

```diff
@@ -512,30 +464,6 @@ spec = do
         let poolId = toText $ getApiT $ p ^. #id
         expectErrorMessage (errMsg403PoolAlreadyJoined poolId) r
 
-    it "STAKE_POOLS_JOIN_01 - \
```

Comment:
removed as it is duplicate of what is in : 
```
it "STAKE_POOLS_JOIN/QUIT - Checking delegation expectations"
```
and we are close to timeout in CI